ONSdigital/eq-survey-runner
# Added accordion to summary page

### What is the context of this PR?
- Added accordions to the summary page
- [Card](https://trello.com/c/aRJ2ZW4M/2077-apply-new-final-summary-design-surveys-with-sections)

### How to review 
- Run alongside schema-validator (onsdigital/eq-schema-validator:eq-2077-apply-summary-add-collapsible-key)
- Run test_section_summary and check section summary and summary are complete, section summary displays without accordion, and summary with accordion.
- Add a collapsible key to 1_0005 schema to test that accordions work.
- Run some other surveys to see that the accordions aren't there.
- Check if any designs are wrong

##
- *Save and Complete button will look wrong until moved to header*
##
### Checklist
